### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by jusnelda**
Made some changes to the login UI.
Let me know what you think!

**Old Login Screen**

old_login
** New Login Screen**

new_login

### CODE REVIEW

File: ios/.symlinks/flutter [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/justine/external/flutter/bin/cache/artifacts/engine
```
Comment (RedTo) [inline ios/.symlinks/flutter]: please put all these files in .gitignore
they will just work on your mac

File: ios/.symlinks/plugins/location [hunk 2]
```diff
@@ -0,0 +1 @@
+/Users/justine/.pub-cache/hosted/pub.dartlang.org/location-1.3.0
```
Comment (RedTo) [inline ios/.symlinks/plugins/location]: same as above

File: ios/.symlinks/plugins/path_provider [hunk 3]
```diff
@@ -0,0 +1 @@
+/Users/justine/.pub-cache/hosted/pub.dartlang.org/path_provider-0.4.0
```
Comment (RedTo) [inline ios/.symlinks/plugins/path_provider]: same as above

File: ios/.symlinks/plugins/url_launcher [hunk 4]
```diff
@@ -0,0 +1 @@
+/Users/justine/.pub-cache/hosted/pub.dartlang.org/url_launcher-3.0.1
```
Comment (RedTo) [inline ios/.symlinks/plugins/url_launcher]: same as above

File: ios/Runner.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -287,21 +286,6 @@
...
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-Runner/Pods-Runner-resources.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
 /* End PBXShellScriptBuildPhase section */
 
 /* Begin PBXSourcesBuildPhase section */
```
Comment (RedTo) [inline ios/Runner.xcodeproj/project.pbxproj]: I'm not quite sure if we should put this file in version control. Look's like a xcode specific one, so maybe it is better to keep locally?!

File: lib/utils/cieStyle.dart [hunk 6]
```diff
@@ -44,7 +44,7 @@ class CiEStyle {
 
   static TextStyle getAppBarTitleStyle(BuildContext context){
     return new TextStyle(
-      fontSize: Theme.of(context).platform == TargetPlatform.iOS ? 17.0 : 20.0,
+      fontSize: Theme.of(context).platform == TargetPlatform.iOS ? 20.0 : 20.0,
```
Comment (RedTo) [inline lib/utils/cieStyle.dart]: could be replace with

fontSize: 20.0,

isn't it?

File: lib/views/login.dart [hunk 7]
```diff
@@ -83,19 +84,20 @@ class LoginFormState extends State<LoginForm> {
                     horizontal: 16.0, vertical: 25.0),
                 children: <Widget>[
                   new Image.asset(StaticVariables.IMAGE_PATH + 'hm_logo.png'),
-                  new Padding(padding: const EdgeInsets.only(top: 25.0)),
+                  new Padding(padding: const EdgeInsets.only(top: 35.0)),
                   new Text(
                     "Courses in English",
-                    style: new TextStyle(color: Colors.red, fontSize: 30.0),
+                    //style: new TextStyle(color: Colors.red, fontSize: 30.0),
```
Comment (RedTo) [inline lib/views/login.dart]: If you like you can remove this comment and also in the lines 97 and 98.

Comment (RedTo) [reply lib/views/login.dart]: and 113 + 114

**Comment by AndiAndreas** [comment]
Nice GUI

**Comment by jusnelda** [comment]
Deleted the unessassary files and deleted comments.

**Comment by jusnelda** [comment]
More changes to the UI:
Course List got an improved look

## Old
old_courselist
## New
new_courselist

File: ios/Runner.xcodeproj/project.pbxproj (hunk 5 above)
Comment (AndiAndreas) [reply ios/Runner.xcodeproj/project.pbxproj]: Have you found something out about that one @jusnelda ? Do we need to keep it? Do you know something about this one @heatsink

**Comment by jusnelda** [comment]
It seems like if i delete that, I can't run the app anymore. Does anyone with a mac have the same experience?

**Comment by heatsink** [comment]
Yeah, the project.pbxproj is necessary for XCode to run the build on the iOS simulator so we should keep it (but the changes don't really need to be in version control).

I think we can just keep this in the commit. In another commit I'll make another to put it in the gitignore, clear the cache, but keep the file.

Also, the UI is looking good! I like the changes!

File: ios/Runner.xcodeproj/project.pbxproj (hunk 5 above)
Comment (RedTo) [reply ios/Runner.xcodeproj/project.pbxproj]: Okay, so let's keep it.

File: .gitignore [hunk 8]
```diff
@@ -3,6 +3,8 @@
 *.iws         // IntelliJ
 .idea/        // IntelliJ
 .DS_Store     // Mac
+.pbxproj // Xcode Mac
```
Comment (RedTo) [inline .gitignore]: Are there any other files in .pbxproj we should keep, or should we remove this line?

File: lib/utils/cieStyle.dart [hunk 9]
```diff
@@ -25,26 +26,27 @@ class CiEStyle {
...
+      fontSize: 14.0,
     );
   }
 
   static TextStyle getMapsTitleStyle() {
     return new TextStyle(
       color: CiEColor.gray,
-      fontWeight: FontWeight.w900,
-      fontSize: 26.0,
+
```
Comment (RedTo) [inline lib/utils/cieStyle.dart]: could be removed

File: lib/utils/cieStyle.dart [hunk 10]
```diff
@@ -11,8 +11,9 @@ class CiEStyle {
 
   static TextStyle getCoursesListTimeStyle() {
     return new TextStyle(
-      fontWeight: FontWeight.bold,
-      color: CiEColor.gray,
+      //fontWeight: FontWeight.bold,
```
Comment (RedTo) [inline lib/utils/cieStyle.dart]: could be removed

File: lib/utils/cieStyle.dart (hunk 6 above)
Comment (RedTo) [reply lib/utils/cieStyle.dart]: Please don't forget to replace this line.

File: .gitignore (hunk 8 above)
Comment (jusnelda) [reply .gitignore]: Deleted this line just to be sure, Mac users can clone the repo and build the app.

File: lib/utils/cieStyle.dart (hunk 9 above)
Comment (jusnelda) [reply lib/utils/cieStyle.dart]: Removed in new commit

**Comment by RedTo** [comment]
Could this branch be deleted?

**Comment by jusnelda** [comment]
I'd like to keep it for a little while longer, to make improvements to timetable and settings page if need be.

**Comment by RedTo** [comment]
Great, sure! Just keep it and happy coding :-)

**Pull request merged**